### ISSUE TITLE
How to create client side page in a sub folder

### ISSUE THREAD

**Issue by sallaadithya** [pnp/pnpjs#2359]
- [ ] Enhancement
- [ ] Bug
- [X] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [3.5.1]
Please specify what version of the SPfx you are using: [1.15.0]
Please specify what version(s) of SharePoint you are targeting: [ SPO ]

### Submission Guidelines
* I need to create a new modern page programmatically in a sub folder, not on the root folder of SitePages library directly.
* in short, I need to create a page with server relative URL, /sites/sitecoll/sitepages/mysubfolder/
* Is there any API I can use to create the same

**Comment by patrick-rodgers** [comment]
This wasn't possible in the API last time we looked into it, or at least we couldn't reverse engineer it. Very happy if anyone has additional insight. We can have another look.

**Comment by juliemturner** [comment]
I've just reviewed again and as far as I can see it still isn't possible in the first party API from the perspective of being able to reverse engineer it, ergo we're not going to be able to add that feature at this time.  You might try creating the page and the using file move to move it into the folder, I've done that with pages before and it works.

**Comment by patrick-rodgers** [comment]
Closing this issue as answered. If you have additional questions or we did not answer your question, please open a new issue, ref this issue, and provide any additional details available. Thank you!
